### ISSUE TITLE
porting cshar-sdk by using .NET Core

### ISSUE THREAD

**Issue by ramogundogdu** [relayr/csharp-sdk#7]
It would be very good if you could port the csharp-sdk to only use the .NET Core so that we can develop applications with the csharp-sdk platform across, like on Linux with Mono...

**Comment by cmatskas** [comment]
I totally agree. I'll look into how effort is required to run against .NET Core and try to get a timeframe out
